### ISSUE TITLE
[Bug]: Extension says email routing setup is not configured properly after removing a subdomain from email routing settings

### ISSUE THREAD

**Issue by Xoconoch** [Curetix/mailflare-extension#34]
### What is this bug about?

I added a subdomain to test the extension, made an alias and deleted afterwards. When I removed the subdomain from the email routing settings and deleted its records, now the extension pops a message warning about email routing settings not being properly configured.

### Steps to reproduce

1. Add a subdomain to email routing settings through the email routing dashboard
2. Create an alias using that subdomain
3. Delete it
4. Remove the subdomain and the created records from the email routing settings

### Additional info

I already tried rerolling the api key. 

What seems to fix it is to re-add the subdomain to the email routing settings.

**Comment by Curetix** [comment]
Yeah, I can reproduce the issue, but I think it has anything to do with MailFlare. After removing the subdomain I get a response from the Cloudflare API that says Email Routing is ready, but not enabled, for the root domain. I'll investigate this further, but it might be a bug in the API.
